src/Microsoft.AspNetCore.Server.Testing/Common/DeploymentParameters.cs

```diff
@@ -73,11 +77,6 @@ public class DeploymentParameters
         public string PublishedApplicationRootPath { get; set; }
 
         /// <summary>
-        /// Passes the --no-source option when publishing.
-        /// </summary>
-        public bool PublishWithNoSource { get; set; }
```

Comment:
no-source is now the default scenario, correct? Will there be an option to publish with source?